Marlin/src/lcd/language/language_en.h

```diff
@@ -138,6 +138,13 @@ namespace LanguageNarrow_en {
   LSTR MSG_PREHEAT_CUSTOM                 = _UxGT("Preheat Custom");
   LSTR MSG_COOLDOWN                       = _UxGT("Cooldown");
 
+  #define _MSG_PREHEAT(N) \
+    LSTR MSG_PREHEAT_##N                  = _UxGT("Preheat ") PREHEAT_## N ##_LABEL; \
+    LSTR MSG_PREHEAT_## N ##_SETTINGS     = _UxGT("Preheat ") PREHEAT_## N ##_LABEL _UxGT(" Conf");
+  #if PREHEAT_COUNT > 1
+    REPEAT_S(2, INCREMENT(PREHEAT_COUNT), _MSG_PREHEAT)
+  #endif
+
```

Comment:
This method of producing strings for DWIN LCD is obsolete, so we shouldn't propagate it to the language files. Instead we do substitution using a string like "Preheat $" where the `$` is replaced by a string supplied in the string emitting function. This block is still retained in `dwin.h` for the benefit of ProUI but it should be getting phased out by @mriscoc at some point soon, if not already.